CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 * **Metrics**
   * [FIX] Set environment variable `MON_TLS_PATH_INGRESS` to ensure correct datasource connection between Grafana
 and Promethues in [Azure Deployment sample](samples/azure-deployment). (Fixes #614)
+  * [CHANGE] Ghostunnel proxy has been removed from project. This was originally done for both Prometheus and Alertmanager. It has
```

Comment:
How about something like:  
[CHANGE] Replaced the ghostunnel sidecar proxy with Grafana's native TLS capabilities and eliminated ghostunnel from the project.